transformations/dyslexia_words_swap/test.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Could you add a few more test cases here, particularly to see how this transformation deals with whitespace?

Reply:
nothing particular with whitespace, only working with words. I am adding a few tests.